functions/alt.h

```diff
@@ -0,0 +1,14 @@
+#ifndef FUNCTIONS_ALTF_H
+#define FUNCTIONS_ALTF_H
+
+// Usage: AltF
+// return value: INTEGER
+// Returns current_alternative for use in ColorSelect<>, TrSelect<> or IntSelect<>
+
+class AltF {
```

Comment:
This should be an SVF, see Int<> for an example.

Reply:
Is that correct or do I need to do the SVFWrapper as well?